integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -6503,6 +6503,18 @@ msgstr ""
 msgid "No changes detected, autosave skipped"
 msgstr "Keine Änderungen vorgenommen, automatisches Speichern übersprungen"
 
+#: cms/views/events/event_form_view.py cms/views/pois/poi_form_view.py
+#, python-format
+msgid ""
+"The slug was changed from '%(user_slug)s' to '%(slug)s', because "
+"'%(user_slug)s' is already used by <a href='%(link)s' class='underline hover:"
+"no-underline'>%(translation)s</a> or one of its previous versions"
+msgstr ""
+"Der URL-Parameter wurde von '%(user_slug)s' zu '%(slug)s' geändert, da "
+"'%(user_slug)s' bereits von <a href='%(link)s' class='underline hover:no-"
+"underline'>%(translation)s</a> oder einer vorherigen Version davon verwendet "
```

Comment:
```suggestion
#, python-brace-format
msgid ""
"The slug was changed from '{user_slug}' to '{slug}', because '{user_slug}' "
"is already used by <a href='{link}' class='underline hover:no-"
"underline'>{translation}</a> or one of its previous versions"
msgstr ""
"Der URL-Parameter wurde von '{user_slug}' zu '{slug}' geändert, da "
"'{user_slug}' bereits von <a href='{link}' class='underline hover:no-"
"underline'>{translation}</a> oder einer vorherigen Version davon verwendet "
```